onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -119,6 +120,8 @@ def __init__(
             else extra_options["QDQOpTypePerChannelSupportToAxis"]
         )
 
+        self.qdq_op_domain = ms_domain if extra_options.get("UseQDQContribOps", False) else None
```

Comment:
Consider inferring from type

Reply:
Can't completely eliminate the `UseQDQContribOps` option because we may want to use the Q/DQ contrib ops for 8-bit integer types as well (for testing or otherwise).

However, I added logic to override the domain to 'com.microsoft' when using 16-bit integer types. I also added a corresponding TODO comment to remove the override when/if ONNX supports 16-bit support.